components/loadercontainer/templates.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Templates
+page_title: LoaderContainer Templates
+description: Templates in the LoaderContainerfor Blazor.
+slug: loadercontainer-templates
+tags: telerik,blazor,loader,container,loadercontainer,templates
+published: True
+position: 10
+---
+
+# LoaderContainer Templates
+
+The LoaderContainerallows you to customize its rendering by using Templates. This article explains the available layout templates for the component.
+
+* [Template](#template)
+
+
+## Template
+
+The `Template` allows you to control the rendering of the LoaderContainer. When you are using the `Template` there will be no panel rendered by default.
+
+This section gives examples that show how to:
```

Comment:
Add screenshots of the results so people could compare with the vanilla component appearance